Localization/it-IT/together-we-are-powerful.po

```diff
@@ -0,0 +1,176 @@
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/together-we-are-powerful/\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20200803\n"
+"Target-Language: it-IT\n"
+"Target-Version: 1.0.0\n"
+"Target-Last-Modified: 20200826\n"
+
+msgid "Together We Are Powerful - Folding@home"
+msgstr "Insieme siamo una forza - Folding@home"
+
+msgid "[Take me home](https://foldingathome.org/home)"
+msgstr "[Vai alla homepage](https://foldingathome.org/home)"
+
+msgid "TOGETHER, WE ARE POWERFUL"
+msgstr "INSIEME, SIAMO UNA FORZA"
+
+msgid "Together, we have created the most powerful supercomputer on the planet, and are using it to help understand SARS-CoV-2/COVID-19 and develop new therapies. We need your help pushing toward a potent, patent-free drug."
+msgstr "Insieme abbiamo creato il più potente supercomputer del pianeta, e lo stiamo usando per studiare il SARS-CoV-2/COVID-19 e per sviluppare nuove terapie. Abbiamo bisogno del tuo aiuto per sviluppare una terapia efficace e libera da brevetti."
+
+msgid "Use your PC to help fight COVID-19."
+msgstr "Usa il tuo PC per aiutarci a combattere il COVID-19."
+
+msgid "[DOWNLOAD FOLDINGATHOME](#downloads)"
+msgstr "[SCARICA FOLDINGATHOME](#downloads)"
+
+msgid "[Available for Windows, Mac, Linux]"
+msgstr "[Disponibile per Windows, Mac, Linux]"
+
+msgid "Progress on the current Sprint 1 to evaluate a batch of potential drugs Started Sun Jul 26 06:31:13 UTC 2020"
+msgstr "Avanzamento dello sprint attuale, con lo scopo di valutare alcune potenziali terapie. Cominciato Domenica 26 Luglio 2020, alle 06:31:13 UTC"
+
+msgid "Campaign Trailer"
+msgstr "Trailer della campagna"
+
+msgid "Play"
+msgstr ""
+
+msgid "Learn about our work on SARS-CoV-2 and COVID-19."
+msgstr "Informazioni sul nostro lavoro sul SARS-CoV-2 e il COVID-19."
+
+msgid "We are focusing the Foldingathome compute power on SARS-CoV-2, the virus that causes COVID-19, and the human proteins the virus interacts with."
+msgstr "Stiamo concentrato il potere computazione del Foldingathome sul SARS-CoV-2, il virus che causa il COVID-19, e sulle proteine umane con cui il virus interagisce."
+
+msgid "First, we uncovered novel protein structures that were previously inaccessible to the research community."
+msgstr "All'inizio abbiamo scoperto nuove strutture proteiche che prima non erano accessibili alla comunità scientifica."
+
+msgid "With your help, we are now screening for potential drugs to target these structures as a part of the COVID Moonshot Collaboration."
+msgstr "Con il tuo aiuto, stiamo cercando potenziali terapie che attacchino queste strutture proteiche, in collaborazione con COVID Moonshot."
+
+msgid "See real-time progress toward new drugs"
+msgstr "Andamento della ricerca in tempo reale"
+
+msgid "Foldingathome Ambassadors"
+msgstr ""
+
+msgid "Satya Nadella"
+msgstr ""
+
+msgid "CEO, Microsoft"
+msgstr ""
+
+msgid "Dr. Lisa Su"
+msgstr ""
```

Comment:
A lot of msgstr are empty, if there is no equivalent I think you may need to keep original text.

Reply:
Is there a guideline on this? The behaviour of gettext is that, when `msgstr` is empty, `msgid` is used, which is to say "if there's no translation, use the original text"

Reply:
Yes, please don't leave any values empty. The system in the end will just replace any instance of the value in `msgid` with the value in `msgstr`. So...if it's empty the resulting page will be incomplete.

Reply:
Fixed